test/core/exceptions.wast

```diff
@@ -0,0 +1,11 @@
+;; Test exception handling support
+
+(module
+  (event)
+  (event (param i32))
```

Comment:
Testing a few more syntactic variations (e.g. with symbolic names) couldn't harm.

Reply:
Done. I did much more extensive testing with my next PR which includes the new EH opcodes. I'll send it for review after this one is merged.
In the meantime I added the relevant tests to this PR as suggested.